### PR TITLE
add openshift pipelines test on rosa hypershift

### PR DESCRIPTION
**Pull request by shakyav**
[LPTOCPCI-1060](https://issues.redhat.com/browse/LPTOCPCI-1060)

### CODE REVIEW

**Comment by shakyav** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.11-openshift-pipelines-ocp4.14-lp-rosa-hypershift-openshift-pipelines-rosa-hypershift

**Comment by shakyav** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.11-openshift-pipelines-ocp4.14-lp-rosa-hypershift-openshift-pipelines-rosa-hypershift

**Comment by shakyav** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.11-openshift-pipelines-ocp4.15-lp-interop-openshift-pipelines-interop-aws

**Comment by shakyav** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.11-openshift-pipelines-ocp4.14-lp-interop-openshift-pipelines-interop-aws

**Comment by shakyav** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.11-openshift-pipelines-ocp4.14-lp-interop-openshift-pipelines-interop-aws

**Comment by shakyav** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.11-openshift-pipelines-ocp4.15-lp-interop-openshift-pipelines-interop-aws

**Comment by shakyav** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.11-openshift-pipelines-ocp4.14-lp-rosa-hypershift-openshift-pipelines-rosa-hypershift

**Comment by shakyav** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.11-openshift-pipelines-ocp4.14-lp-rosa-hypershift-openshift-pipelines-rosa-hypershift

**Comment by shakyav** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.11-openshift-pipelines-ocp4.14-lp-interop-openshift-pipelines-interop-aws

**Comment by shakyav** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.11-openshift-pipelines-ocp4.14-lp-rosa-hypershift-openshift-pipelines-rosa-hypershift

**Comment by shakyav** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.11-openshift-pipelines-ocp4.14-lp-rosa-hypershift-openshift-pipelines-rosa-hypershift

File: ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.11__openshift-pipelines-ocp4.14-lp-rosa-hypershift.yaml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+resources:
+  '*':
+    limits:
+      memory: 8Gi
+    requests:
+      cpu: "1"
+      memory: 4Gi
+tests:
+- as: openshift-pipelines-rosa-hypershift
+  cron: 0 6 25 12 *
```
Comment (oharan2) [inline ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.11__openshift-pipelines-ocp4.14-lp-rosa-hypershift.yaml:42]:
```suggestion
  cron: 0 6 25 10 *
```

File: ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.11__openshift-pipelines-ocp4.14-lp-rosa-hypershift.yaml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+                {"step": "aws-deprovision*", "failure_type": "all", "classification": "Infrastructure Deprovisioning - AWS", "jira_project": "LPINTEROP", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["!default"]},
+                {"step": "cucushift-hypershift-*", "failure_type": "all", "classification": "Health Check - Cluster", "jira_project": "LPINTEROP", "group": {"name": "lp-tests", "priority": 2}, "jira_additional_labels": ["!default"]},
+                {"step": "gather-*", "failure_type": "pod_failure", "classification": "Must Gather - Cluster", "jira_project": "LPINTEROP", "group": {"name": "cluster", "priority": 3}, "jira_additional_labels": ["!default"]},
+                {"step": "gather-*", "failure_type": "test_failure", "classification": "NONE", "jira_project": "NONE", "ignore": "true"},
+                {"step": "openshift-pipelines-install-and-tests", "failure_type": "pod_failure", "classification": "Test Failure", "jira_project": "LPINTEROP", "group": {"name": "cluster", "priority": 3},"jira_additional_labels": ["!default"]},
```
Comment (oharan2) [inline ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.11__openshift-pipelines-ocp4.14-lp-rosa-hypershift.yaml:55]:
```suggestion
                {"step": "openshift-pipelines-install-and-tests", "failure_type": "pod_failure", "classification": "Test Failure", "jira_project": "LPINTEROP", "group": {"name": "lp-tests", "priority": 1},"jira_additional_labels": ["!default"]},
```

File: ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.11__openshift-pipelines-ocp4.14-lp-rosa-hypershift.yaml [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+                {"step": "openshift-pipelines-install-and-tests", "failure_type": "test_failure", "classification": "Test Failure", "jira_project": "SRVKP", "group": {"name": "cluster", "priority": 3}, "jira_additional_labels": ["!default"]},
```
Comment (oharan2) [inline ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.11__openshift-pipelines-ocp4.14-lp-rosa-hypershift.yaml:56]:
```suggestion
                {"step": "openshift-pipelines-install-and-tests", "failure_type": "test_failure", "classification": "Test Failure", "jira_project": "SRVKP", "group": {"name": "lp-tests", "priority": 1}, "jira_additional_labels": ["!default"]},
```

File: ci-operator/step-registry/openshift-pipelines/install-and-tests/openshift-pipelines-install-and-tests-ref.yaml [hunk 4]
```diff
@@ -11,5 +11,9 @@ ref:
       default: latest
       documentation: |-
         OLM channel selected for openshift-pipelines operator.
+    - name: ROSA
```
Comment (oharan2) [inline ci-operator/step-registry/openshift-pipelines/install-and-tests/openshift-pipelines-install-and-tests-ref.yaml:14]: I think this env var is eventually unused, maybe your first thought was to use it [in the login section](https://github.com/openshift/release/pull/47988/files#diff-7b19e927a3d5d53bfcef2d8421b10f3858470539ca43d2a476765a3fbf2719dfR23)?

**Review by oharan2 — CHANGES_REQUESTED** [review]

File: ci-operator/step-registry/openshift-pipelines/install-and-tests/openshift-pipelines-install-and-tests-ref.yaml (hunk 4 above)
Comment (shakyav) [reply ci-operator/step-registry/openshift-pipelines/install-and-tests/openshift-pipelines-install-and-tests-ref.yaml:14]: I noticed that, just wanted to leave it for future purpose, but you are right, I should just remove it for now

**Comment by shakyav** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.11-openshift-pipelines-ocp4.14-lp-rosa-hypershift-openshift-pipelines-rosa-hypershift

**Review by calebevans — APPROVED** [review]
/lgtm

**Comment by shakyav** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.11-openshift-pipelines-ocp4.14-lp-rosa-hypershift-openshift-pipelines-rosa-hypershift

**Comment by shakyav** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.11-openshift-pipelines-ocp4.14-lp-interop-openshift-pipelines-interop-aws

**Comment by shakyav** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.11-openshift-pipelines-ocp4.14-lp-rosa-hypershift-openshift-pipelines-rosa-hypershift

**Comment by shakyav** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.11-openshift-pipelines-ocp4.14-lp-rosa-hypershift-openshift-pipelines-rosa-hypershift

**Comment by shakyav** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.11-openshift-pipelines-ocp4.14-lp-rosa-hypershift-openshift-pipelines-rosa-hypershift

**Comment by shakyav** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.11-openshift-pipelines-ocp4.14-lp-rosa-hypershift-openshift-pipelines-rosa-hypershift

**Review by calebevans — APPROVED** [review]
/lgtm

**Pull request merged**